build/run-tasks.js

```diff
@@ -0,0 +1,151 @@
+const colour = require('nodemon/lib/utils/colour')
+const del = require('del')
+const fs = require('fs')
+const fse = require('fs-extra')
+const path = require('path')
+const chokidar = require('chokidar')
+
+const buildConfig = require('./config.json')
+const { paths } = buildConfig
+
+const appSassPath = path.join(paths.assets, 'sass')
+const docsSassPath = path.join(paths.docsAssets, 'sass')
+const v6SassPath = path.join(paths.v6Assets, 'sass')
+
+const appCssPath = path.join(paths.public, 'stylesheets')
+const v6CssPath = path.join(paths.public, 'v6', 'stylesheets')
+
+const v6PublicPath = path.join(paths.public, 'v6')
+
+function generateAssets () {
+  clean()
+  sassExtensions()
+  sass(appSassPath, appCssPath)
+  sass(docsSassPath, appCssPath)
+  sass(v6SassPath, v6CssPath)
+  copyAssets(paths.assets, paths.public)
+  copyAssets(paths.docsAssets, paths.public)
+  copyAssets(paths.v6Assets, v6PublicPath)
+}
+
+function runTasks () {
```

Comment:
I wonder if we can give this a more descriptive name 🤔 

`buildWatchAndServe` maybe?

Reply:
Renamed as suggested